spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -10,11 +10,12 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-stream-binders-parent</artifactId>
-		<version>1.0.0.BUILD-SNAPSHOT</version>
+		<version>${project.version}</version>
 	</parent>
 
 	<properties>
 		<kafka.version>0.8.2.2</kafka.version>
```

Comment:
I believe these version properties need to move to the parent, along with the <dependencyManagement> statements, leaving only <dependencies> in modules.